src/Canvas.cc

```diff
@@ -958,9 +887,7 @@ Canvas::createCairoContext() {
  * Construct an Error from the given cairo status.
  */
 
-Local<Value>
-Canvas::Error(cairo_status_t status) {
-  return Exception::Error(Nan::New<String>(cairo_status_to_string(status)).ToLocalChecked());
+Napi::Error
+Canvas::CairoError(cairo_status_t status) {
+  return Napi::Error::New(env, cairo_status_to_string(status));
 }
-
-#undef CHECK_RECEIVER
```

Comment:
Probably keep this line? The macro's name no longer reflects what it does either.

Reply:
The name still reflects the purpose, not so much the way it's done anymore.

I think I'm going to wait until another node-addon-api release so we can just delete the checks.

Reply:
> I think I'm going to wait until another node-addon-api release so we can just delete the checks.

`node-addon-api` 7.0 was released with my fix, but they also bumped the minimum node version to 16. We can remove like 100 lines and this PR is marked as semver-major anyways, so I'm going to go forward with it...